data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/dao/ScalarDBManager.java

```diff
@@ -0,0 +1,68 @@
+package com.scalar.db.dataloader.core.dataimport.dao;
+
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.api.DistributedTransactionAdmin;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.service.StorageFactory;
+import com.scalar.db.service.TransactionFactory;
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+/**
+ * A manager to retrieve the various ScalarDB managers based on the running mode
+ *
+ * @author Yves Peckstadt
+ */
+public class ScalarDBManager {
```

Comment:
Per [this comment](https://github.com/scalar-labs/scalardb/pull/2417#discussion_r1917819691), this class has been split, right? Git operation related issue?

Reply:
This PR was created before (which may had the file as I was creating multiple PRs for all changes). I will remove the file.
Thank you.